report_thesis/src/sections/pyhat_contribution.tex

```diff
@@ -0,0 +1,22 @@
+\section{PyHAT Contribution}\label{sec:pyhat_contribution}
+As part of our research, we have made several contributions to \gls{pyhat}, which offers a user-friendly interface designed for performing machine learning and data analysis tasks specifically for hyperspectral data.
+This collaboration was initiated through a series of discussions with two members \gls{usgs} responsible for managing \gls{pyhat}, wherein we identified mutual challenges and opportunities for integrating our solutions into the library.
+
+The largest contribution involved the integration of an automatic outlier detection method into the library's \gls{pls} implementation. 
```

Comment:
```suggestion
The largest contribution involved the integration of an automatic outlier detection method into \gls{pyhat}.
```

Reply:
Wasn't specifically for the PLS implementation. It's a general method for outlier removal. It just uses PLS.